### PR TITLE
Add new translations

### PR DESCRIPTION
**Pull request by sheilagomes**
Add Catalan, German, Italian and Romanian translations. Tracked in task [LOC-9765](https://vtex-dev.atlassian.net/browse/LOC-9765).

[LOC-9765]: https://vtex-dev.atlassian.net/browse/LOC-9765?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/vtex-my-subscriptions-portal-connector/messages/de.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "store/subscription.menu.link": "Abonnements"
```
Comment (camilavcoutinho) [inline apps/vtex-my-subscriptions-portal-connector/messages/de.json:2]: é isso mesmo ? 🤔

Comment (sheilagomes) [reply apps/vtex-my-subscriptions-portal-connector/messages/de.json:2]: Eu copiei daqui: https://github.com/vtex/my-subscriptions/blob/600989993db45464dd6299cff0891288c89f026d/apps/vtex-my-subscriptions-3/messages/de.json#L2

**Comment by Fbenitez97** [comment]
Hello @camilavcoutinho! 😄 

Do you think this can be published already? 

Thanks for your work @sheilagomes!

**Review by lschlee — APPROVED** [review]

**Comment by sheilagomes** [comment]
> Hello @camilavcoutinho! smile
> 
> Do you think this can be published already?
> 
> Thanks for your work @sheilagomes!

We only needed one more approval and we've just got it, so I'll proceed with the publishing.

**Pull request merged**